tpke/benches/tpke.rs

```diff
@@ -299,12 +299,77 @@ pub fn bench_share_encrypt_decrypt(c: &mut Criterion) {
     }
 }
 
+pub fn bench_recover_share_at_point(c: &mut Criterion) {
+    let mut group = c.benchmark_group("Recover Share at Point Benchmark");
+    // Set up test conditions
+    let rng = &mut StdRng::seed_from_u64(0);
+    let msg_size = MSG_SIZE_CASES[0];
+
+    for &shares_num in NUM_SHARES_CASES.iter() {
+        let mut setup = SetupSimple::new(shares_num, msg_size, rng);
+        let threshold = shares_num * 2 / 3;
+        let selected_participant = setup.contexts.pop().unwrap();
+        let x_r = selected_participant
+            .public_decryption_contexts
+            .last()
+            .unwrap()
+            .domain;
+        let mut remaining_participants = setup.contexts;
+        for p in &mut remaining_participants {
+            p.public_decryption_contexts.pop();
+        }
+        group.bench_with_input(
+            BenchmarkId::new("Recover Share at Point", shares_num),
+            &shares_num,
+            |b, &_shares_num| {
+                let mut rng = rand::rngs::StdRng::seed_from_u64(0);
+                b.iter(|| {
+                    black_box(recover_share_at_point::<E>(
+                        &remaining_participants[..shares_num],
+                        threshold,
+                        &x_r,
+                        &mut rng,
+                    ));
```

Comment:
```suggestion
                    black_box(
                        let _ = recover_share_at_point::<E>(
                            &remaining_participants[..shares_num],
                            threshold,
                            &x_r,
                            &mut rng);
                    );
```
The return value of `recover_share_at_point` must be used, either by assignment to a variable or by adding `;` at the end of the function call. Otherwise, we're forcing the scope to return the return value instead of returning just `()`.